kaldb/src/main/java/com/slack/kaldb/metadata/cache/CacheSlotMetadataStore.java

```diff
@@ -15,12 +16,14 @@ public class CacheSlotMetadataStore extends KaldbPartitioningMetadataStore<Cache
    * Initializes a cache slot metadata store at the CACHE_SLOT_ZK_PATH. This should be used to
    * create/update the cache slots, and for listening to all cache slot events.
    */
-  public CacheSlotMetadataStore(AsyncCuratorFramework curatorFramework) throws Exception {
+  public CacheSlotMetadataStore(AsyncCuratorFramework curatorFramework, MeterRegistry meterRegistry)
```

Comment:
A large part of this diff if adding `meterRegistry` parameter. It would be far more easier to review the diff if `meterRegistry` change is done in a separate PR.